### ISSUE TITLE
Could you share the train log for the baseline model?

### ISSUE THREAD

**Issue by kwonyoung9120** [MCG-NJU/SparseOcc#21]
Thank you for your great work! 

When I evaluate your model, the model trained performed about 29.x mIoU with old metric. 
However, when I trained same model with loading your pretrained model, the total loss starts about 35~40, and after training 48 epochs, the mIoU drops (about 15, loss : 18.x). How much loss should be reproduced when I load your weight? Maybe I missed something.

**Comment by afterthat97** [comment]
Our logs are saved in wandb:

https://api.wandb.ai/links/liuhs-team/s0o1ph9m

**Comment by afterthat97** [comment]
Sorry for the late reply

**Comment by kwonyoung9120** [comment]
Sorry for late review, and thank you for your logs! Close the issue :)

**Comment by kwonyoung9120** [comment]
Thanks for the updates and congrats for ECCV! 

When I run your code with panoptic segmentation, the training loss with reload your trained model records about to 30.x, is it reasonable? It seems some of the predictions are not completed as below. But I don't think this is the visualization error..(Top : segmentation, Bottom : panoptic) (validation 0004) 

![0004-01-sem](https://github.com/user-attachments/assets/3f148567-027f-4864-ab07-7252b7bb298d)
![0004-01-inst](https://github.com/user-attachments/assets/ab684955-b73b-4670-b651-5f60fdc46fe2)

**Comment by kwonyoung9120** [comment]
Ah sorry that was my mistake, I brought the segmentation results from the voxel decoder. Closing this issue.
